securedrop/tests/test_journalist.py

```diff
@@ -744,9 +748,11 @@ def test_logo_upload_with_valid_image_succeeds(self):
 
         try:
             self._login_admin()
-
+            # Create 1px * 1px 'white' PNG file from its base64 string
             form = journalist_app.forms.LogoForm(
-                logo=(StringIO('imagedata'), 'test.png')
+                logo=(BytesIO(base64.decodestring
+                ("iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQ"
```

Comment:
For the other reviewers, this was something I gave @aydwi out of band, and that is definitely a 1x1 white pixel.

Reply:
Yes Thanks @heartsucker. I added a comment to clarify that to anyone reading the code as well. Also, I think including an image whose size is greater than 500px can also help test the resize function, although the base64 string will be very large.